<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindowPage</class>
 <widget class="QWidget" name="ChatWindowPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>692</height>
   </rect>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>851</width>
     <height>611</height>
    </rect>
   </property>
   <widget class="QWidget" name="splitter" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QTabWidget" name="tab">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>541</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="widget_4">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>701</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QListView" name="userView">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>161</width>
       <height>391</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnect">
     <property name="geometry">
      <rect>
       <x>595</x>
       <y>420</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
     </property>
     <property name="text">
      <string>Leave</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="hide3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>671</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>179</x>
       <y>40</y>
       <width>241</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hide1">
      <property name="spacing">
       <number>7</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font-weight: bold;
font-size: 20px;</string>
        </property>
        <property name="text">
         <string>HTH World Chat</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="editPseudo">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>210</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nickname:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonConnect">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>275</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 2px;
border-color: #34bcaa;
border-radius: 10px;
font-size: 11px;</string>
     </property>
     <property name="text">
      <string>Click to Connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>142</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Connect to #HTHWorld TrollBox&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>521</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;To Chat on Web, Go to &lt;a href=&quot;https://webchat.freenode.net/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://webchat.freenode.net/&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;Enter your Nick&lt;/p&gt;&lt;p&gt;Enter #HTHWorld as Channel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCloseTab">
   <property name="text">
    <string>Fermer l'onglet</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
